macros.in

```diff
@@ -715,7 +715,8 @@ package or when debugging this package.\
   RPM_ARCH=\"%{_arch}\"\
   RPM_OS=\"%{_os}\"\
   RPM_BUILD_NCPUS=\"%{_smp_build_ncpus}\"\
-  export RPM_SOURCE_DIR RPM_BUILD_DIR RPM_OPT_FLAGS RPM_ARCH RPM_OS RPM_BUILD_NCPUS\
+  RPM_SPECPARTS_DIR=\"%{u2p:%{specpartsdir}}\"\
```

Comment:
Rebase on top of #2267 and drop the u2p here too. Sorry for not spotting this earlier.

Reply:
Done.